paasta_tools/tron_tools.py

```diff
@@ -723,3 +723,14 @@ def list_tron_clusters(service: str, soa_dir: str = DEFAULT_SOA_DIR) -> List[str
         if cluster_re_match is not None:
             clusters.append(cluster_re_match.group(1))
     return clusters
+
+
+def get_tron_api_endpoints(cluster: str):
```

Comment:
I think this should just be `get_tron_api_endpoint` (not plural) if it only returns one